iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -189,6 +193,10 @@ public enum ErrorCode
         [EditorBrowsable(EditorBrowsableState.Never)]
         PartitionNotFound = 503003,
 
+        /// <summary>
+        /// The operation failed because the module cannot be found by IoT Hub. The module is either not registered or disabled. May be thrown by operations such as 
```

Comment:
Is there a concept of module registration at all?

Reply:
Yes, you register modules to a device in your IoT Hub much the same way you register devices to your IoT Hub